Zend/tests/enum/scalar-duplicate-int.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+Scalar enums reject duplicate int values
+--FILE--
+<?php
+
+enum Foo: int {
+    case Bar = 0;
+    case Baz = 0;
+}
+
+?>
+--EXPECTF--
+Fatal error: Duplicate enum case value in %s on line %s
```

Comment:
Maybe include the value in the message?

Reply:
Strings could include be long, be heredoc or contain newlines so embedding the string in the error message could look ugly. In the match RFC we avoided printing the values for this reason. We could just do it for ints but that would be inconsistent.

Reply:
Right. I guess we could do something like `Duplicate value for enum cases %s and %s` but maybe not worthwhile.